articles/api-auth/using-rules/client-credentials.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Describes using rules with Client Credentials Grant.
+---
+
+# Using Rules with Client Credentials Grants
+
+You can now add [rules](/rules) into the [client credentials](/api-auth/grant/client-credentials) exchange pipeline where you exchange a `client_id` and `secret` for an `access_token`.
+
+## Prior to Beginning Your Configuration
+
+<%= include('./_includes/_prerequisites.md') %>
+
+## Creating the Rule
+
+**Note**: You can only create one rule, which will then be executed for **all** clients and APIs.
+
+### 1. Create the Rule For Use with Webtasks
+
+<%= include('./_includes/_create-rule', {
+	  args: 'client, scope, audience, context, cb'
```

Comment:
@rolodato @hernanhht I talked with Geoff, the problem was that `wt` was creating my webtask at my 'default' profile (which, for me, points to webtask.io). I added at the end of the `wt create` command the: `-p mpaktiti-default` (where `mpaktiti` is my tenant name) and it worked. So if you already have webtask profiles you should use:
```
SECRET=$(openssl rand 32 -base64) && \
wt create myrule.js \
  --meta wt-compiler=auth0-ext-compilers/client-credentials-exchange \
  --meta auth0-extension=runtime \
  --meta auth0-extension-name=credentials-exchange \
  --meta auth0-extension-secret=$SECRET \
  --secret auth0-extension-secret=$SECRET \
  -p "TENANT-default"
```

I'll add a note at the docs.

@rolodato please try and reproduce once more, and if it does not work for you still let me know. Thanks!